### ISSUE TITLE
All tabs with same width

### ISSUE THREAD

**Issue by alejandrogiubel** [afonsocraposo/buttons_tabbar#45]
Hi great plugin, very useful. Will be nice add a property to define same width to all tabs.

**Comment by Ruqaiya-NextgenI** [comment]
has anyone tried, it's unselected label color is not changing on tapping

**Comment by madhumitha-scholarly** [comment]
> Hi great plugin, very useful. Will be nice add a property to define same width to all tabs.

Agreed, great plugin but i have a requirement of equal width tabs and sadly there is no option for that

**Comment by madhumitha998** [comment]
> Hi great plugin, very useful. Will be nice add a property to define same width to all tabs.

Agreed, great plugin but i have a requirement of equal width tabs and sadly there is no option for that
